src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Slider from "react-slick";
+import "../../../node_modules/slick-carousel/slick/slick.css"; 
+import "../../../node_modules/slick-carousel/slick/slick-theme.css";
+import './Carousel.css'
+import Hero from '../Hero/Hero.js'
+
+const Carousel = ({movies}) => {
+  const featuredFilms = movies.slice(10, 15).map(movie => {
+    return (
+      <Hero 
+        title={movie.title} 
+        image={movie.backdrop_path} 
+        key={movie.id}
+        id={movie.id}
+      />
+    )
+  })
+
+  const settings = {
+    dots: true,
+    infinite: true,
+    speed: 2000,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    // autoplaySpeed: 7000,
+    // autoplay: true
```

Comment:
As we discussed, the autoplay is getting annoying during development, but I think it will be cool to turn this back on at the end.